From c4c3d67a5fd51227f0a7043fbaa9212a7c41fd05 Mon Sep 17 00:00:00 2001
From: Fugang Duan <b38611@freescale.com>
Date: Wed, 6 May 2015 20:02:30 +0800
Subject: [PATCH 0524/1221] MLK-10836 ARM: dts: imx6ul-pinfunc: correct one
 uart pin setting

Comparing the latest iomux pin setting from PEx team, found one
uart pin setting is not correct. This patch just correct it.

Signed-off-by: Fugang Duan <B38611@freescale.com>
[zou:Original patch taken from
git.freescale.com/imx/fsl-arm-yocto-bsp.git-b imx-3.14.52-1.1.0_ga]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 arch/arm/boot/dts/imx6ul-pinfunc.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ul-pinfunc.h b/arch/arm/boot/dts/imx6ul-pinfunc.h
index 64eed24..20b04ce 100644
--- a/arch/arm/boot/dts/imx6ul-pinfunc.h
+++ b/arch/arm/boot/dts/imx6ul-pinfunc.h
@@ -891,7 +891,7 @@
 #define	MX6UL_PAD_CSI_DATA02__EIM_AD02                           	0x01EC 0x0478 0x0000 4 0
 #define	MX6UL_PAD_CSI_DATA02__GPIO4_IO23                         	0x01EC 0x0478 0x0000 5 0
 #define	MX6UL_PAD_CSI_DATA02__SAI1_RX_SYNC                       	0x01EC 0x0478 0x0000 6 0
-#define	MX6UL_PAD_CSI_DATA02__UART5_DCE_RTS                      	0x01EC 0x0478 0x0640 8 3
+#define	MX6UL_PAD_CSI_DATA02__UART5_DCE_RTS                      	0x01EC 0x0478 0x0640 8 5
 #define	MX6UL_PAD_CSI_DATA02__UART5_DTE_CTS                      	0x01EC 0x0478 0x0000 8 0
 #define	MX6UL_PAD_CSI_DATA03__CSI_DATA05                         	0x01F0 0x047C 0x0000 0 0
 #define	MX6UL_PAD_CSI_DATA03__USDHC2_DATA3                       	0x01F0 0x047C 0x0688 1 0
-- 
1.7.5.4

